django_ipam/base/admin.py

```diff
@@ -22,6 +22,8 @@ class AbstractSubnetAdmin(TimeReadonlyAdminMixin, ModelAdmin):
     change_form_template = "admin/django-ipam/subnet/change_form.html"
     change_list_template = "admin/django-ipam/subnet/change_list.html"
     app_name = "django_ipam"
+    list_display = ('subnet', 'name', 'master_subnet', 'description')
```

Comment:
I think we should show the name first, because different subnets are assigned to different areas of the networks which are used for different purposes and humans recognize names more easily than ip address ranges.